wordlistctl.py

```diff
@@ -267,12 +268,12 @@ def add_argument(self, *args, **kwargs):
         def format_usage(self):
 
             # Use user-defined usage message
-            if ("usage" in self.program):
+            if "usage" in self.program:
                 prefix = "Usage: "
                 wrapper = textwrap.TextWrapper(width=self.width)
                 wrapper.initial_indent = prefix
                 wrapper.subsequent_indent = len(prefix) * " "
-                if (self.program["usage"] == "" or str.isspace(self.program["usage"])):
+                if self.program["usage"] == "" or str.isspace(self.program["usage"]):

```

Comment:
If you are trying to kill trailing/leading whitespaces, you can just `strip()` the instance instead of this check. If not : 

```python
if not self.program["usage"].replace(" ",""):
```

Reply:
kind of done i guess